src/server/ExpressServer.ts

```diff
@@ -38,7 +38,6 @@ export class ExpressServer {
     }
 
     public async init(): Promise<void> {
```

Comment:
We cannot merge it because the server is started by svelte in dev mode. `npm run server` has just the socket.

Lemme fix the dev mode not working in this PR.